src/modules/Servicedomain/html_client/mod_servicedomain_manage.html.twig

```diff
@@ -1,325 +1,322 @@
+                            <a class="fs-4" target="_blank" href="http://{{ service.domain }}">{{ service.domain }}</a>
                         </div>
-                    </div>
-
-                </div>
-                <div class="tab-pane" id="privacy">
-                    <h2>{{ 'Domain Privacy Settings'|trans }}</h2>
-                    <div class="block">
-                        <p class="alert alert-info">{{ 'If you would like to hide domain contact information which is shown on WHOIS you can enable privacy protection. Once domain privacy protection is enabled no one will know who registered this service. And once you decide to disable privacy protection, information that is listed above on "Update Domain Contact Details" section will be seen on domain WHOIS again.'|trans }}</p>
-                        <div class="control-group">
-                            <div class="controls">
-                                <button class="btn btn-primary" type="button" id="domain-disable-pp" {% if service.privacy == 0 %}style="display:none;"{% endif %}>{{ 'Disable Privacy protection'|trans }}</button>
-                                <button class="btn btn-primary" type="button" id="domain-enable-pp" {% if service.privacy == 1 %}style="display:none;"{% endif %}>{{ 'Enable Privacy protection'|trans }}</button>
-                            </div>
+                        <div class="d-flex flex-column">
+                            <span class="fs-6 text-muted">{{ 'Expires on'|trans }}</span>
+                            <span class="fs-4">{{ service.expires_at|format_date(pattern="MMM dd, yyyy") }}</span>
+                        </div>
+                        <div class="d-flex flex-column">
+                            <span class="fs-6 text-muted">{{ 'Registered on'|trans }}</span>
+                            <span class="fs-4">{{ service.registered_at|format_date(pattern="MMM dd, yyyy") }}</span>
                         </div>
                     </div>
                 </div>
+            </div>
 
-                <div class="tab-pane" id="nameservers">
-                    <h2>{{ 'Update Nameservers'|trans }}</h2>
-                    <div class="block">
-                        <p class="alert alert-info">{{ 'If you would like to host this domain with another company you can update nameservers.'|trans }}</p>
-                        <form action="" method="post" id="update-nameservers" class="form-horizontal">
+            {# Domain protection #}
+            <div class="card-body tab-pane" id="protect-tab-pane" role="tabpanel" tabindex="0">
+                <div class="mb-3">
+                    <h5 class="mt-1 mb-1">{{ 'Domain Protection'|trans }}</h5>
+                    <span class="small text-secondary">{{ 'Domain locking is a security enhancement to prevent unauthorized transfers of your domain to another registrar or web host by "locking" your domain name servers.'|trans }}</span>
```

Comment:
Text should be: {{ 'Domain locking is a security enhancement to prevent unauthorized transfers of your domain to another registrar. This will require you to unlock the domain before transferring it to another registrar. This is an additional security measure to protect against abuse, for example if your transfer code has been compromised.'|trans }}

https://www.icann.org/resources/pages/locked-2013-05-03-en